packages/blockchain-wallet-v4-frontend/src/modals/Swap/EnterAmount/Checkout/validation.tsx

```diff
@@ -17,6 +17,7 @@ export const getMaxMin = (
   BASE: SwapAccountType,
   COUNTER: SwapAccountType
 ) => {
+  // eslint-disable-next-line default-case
```

Comment:
can we just add `default: break` and then not disable the rule?

Reply:
sure, will change